### PR TITLE
fix: fixes silly mistake with testing error message in location search

### PR DESCRIPTION
**Pull request by mikaelbr**

### CODE REVIEW

File: src/location-search/LocationSearch.tsx [hunk 1]
```diff
@@ -182,7 +180,7 @@ const LocationSearch: React.FC<Props> = ({
         </View>
       </View>
 
-      {true && (
+      {error && (
```
Comment (rikkekantega) [inline src/location-search/LocationSearch.tsx:183]: How did this even pass .... Oh well

**Review by rikkekantega — APPROVED** [review]
Please merge

File: src/location-search/LocationSearch.tsx (hunk 1 above)
Comment (mikaelbr) [reply src/location-search/LocationSearch.tsx:183]: It was me updating the code from your code review and pushing silly code. Figured it didn't get reviewed after the last commit.

Comment (rikkekantega) [reply src/location-search/LocationSearch.tsx:183]: No, didn't see through changes past my last comment I think :woozy_face:

**Pull request merged**